src/tactic/lean_core_docs.lean

```diff
@@ -109,11 +109,6 @@ add_tactic_doc
 `by_cases p` splits the main goal into two cases, assuming `h : p` in the first branch, and
 `h : ¬ p` in the second branch. You can specify the name of the new hypothesis using the syntax
 `by_cases h : p`.
```

Comment:
```suggestion
`by_cases h : p`.

If `p` is not already decidable, `by_cases` will use the instance `classical.prop_decidable p`.
```
Maybe this should be added to the doc string in core Lean as well.

Reply:
Cool, I wasn't sure whether it was worth mentioning that but let's add it in and see what other reviewers think 🙂